packages/lodestar-cli/src/depositContract/depositData.ts

```diff
@@ -35,7 +36,7 @@ export function decodeEth1TxData(
   // Sanity check
   const depositDataRoot = config.types.DepositData.hashTreeRoot(depositData);
   if (depositDataRoot != root)
-    throw Error("deposit data root mismatch");
+    throw new YargsError("deposit data root mismatch");
```

Comment:
This is not an "expected" error, I would throw a regular error

Reply:
done